### ISSUE TITLE
progress bar positioning - (issue and fix)

### ISSUE THREAD

**Issue by srikanthpagadala** [xtk/X#94]
# ISSUE AND FIX
#### Issue Description:

I use XTK with GWT. I have had issues with positioning my progress bar in my widgets from the time I started using XTK (about 8 months). I have been also discussing this issue with Daniel since May on stackoverflow.

http://stackoverflow.com/questions/10717282/progress-bar-positioning/12149625#12149625

Today I decided to debug:
#### Fix:

In X/ui/progressbar.js, line 181 and 182.

Before:

  pbElement.style.top = (this._parent.clientHeight - 5) / 2 ;
  pbElement.style.left = (this._parent.clientWidth - 100) / 2;

After:

  pbElement.style.top = (this._parent.clientHeight - 5) / 2 + "px";
  pbElement.style.left = (this._parent.clientWidth - 100) / 2 + "px";

Basically, simply add "px" at the end of the two lines. This solves all the progress bar positioning problems.

Getting support for overload-able the CSS class xtk-progress-bar, is an added bonus in the quest to solve this issue.
